### PR TITLE
Adapt to projects build changes

### PR DESCRIPTION
**Pull request by Maxez**
## Motivation and Context

[JIRA: PP-17](https://jira.phoenix-rtos.com/browse/PP-17)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
The test runner supports project `TARGET`s only (e.g. armv7m7-imxrt106x-project).

## How Has This Been Tested?

- [x] Already covered by automatic testing.
- [ ] New test added: (add PR link here).
- [ ] Tested by hand on: (list targets here).

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing linter checks and tests passed.
- [x] My changes generate no new compilation warnings for any of the targets.

## Special treatment

- [x] This PR needs additional PRs to work (list the PRs, preferably in merge-order).
https://github.com/phoenix-rtos/phoenix-rtos-project/pull/413
- [ ] I will merge this PR by myself when appropriate.

### CODE REVIEW

**Comment by jsarzy** [comment]
Please fix test cases for `target` keyword. Change target `ia32-generic` in the input of the test [here](https://github.com/phoenix-rtos/phoenix-rtos-tests/blob/d6fe2a7f50bcb80a3709fab878ce657e7493b51d/trunner/test/test_config.py#L159) and [here](https://github.com/phoenix-rtos/phoenix-rtos-tests/blob/d6fe2a7f50bcb80a3709fab878ce657e7493b51d/trunner/test/test_config.py#L169) (second test passes but the result can be false positive)

File: psh/test.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 test:
     targets:
-      #TODO: remove when zynq7000-qemu target will stop being experimental
-      include: [armv7a9-zynq7000-qemu]
+      #TODO: remove when armv7a9-zynq7000-qemu and riscv64-generic-qemu targets will stop being experimental
+      include: [armv7a9-zynq7000-qemu, riscv64-generic-qemu]
```
Comment (damianloew) [inline psh/test.yaml:4]: Did You run these tests for `riscv64-generic-qemu`? I am curious about results, if everything works fine I will enable it in CI.

**Comment by Maxez** [comment]
> Please fix test cases for `target` keyword. Change target `ia32-generic` in the input of the test [here](https://github.com/phoenix-rtos/phoenix-rtos-tests/blob/d6fe2a7f50bcb80a3709fab878ce657e7493b51d/trunner/test/test_config.py#L159) and [here](https://github.com/phoenix-rtos/phoenix-rtos-tests/blob/d6fe2a7f50bcb80a3709fab878ce657e7493b51d/trunner/test/test_config.py#L169) (second test passes but the result can be false positive)

Done, changed `target` from `ia32-generic` to `ia32-generic-qemu`.

File: psh/test.yaml (hunk 1 above)
Comment (Maxez) [reply psh/test.yaml:4]: Yes, I did. All `psh` tests pass except for the `auth` test - that's the reason why we shouldn't enable `riscv64-generic-qemu` target quite yet.

**Pull request merged**